test/integration/targets/docker_host_facts/tasks/test_host_facts.yml

```diff
@@ -0,0 +1,274 @@
+---
+- name: Create random container/volume name
+  set_fact:
+    cname: "{{ 'ansible-ctest-%0x' % ((2**32) | random) }}"
+    vname: "{{ 'ansible-vtest-%0x' % ((2**32) | random) }}"
```

Comment:
Please use `ansible-test-` as the prefix for both, not anything else. Otherwise, the cleanup code in `setup_docker` will not find these.